### ISSUE TITLE
Parallelize CV calculations

### ISSUE THREAD

**Issue by ryanbanderson** [USGS-Astrogeology/PySAT_Point_Spectra_GUI#155]
It should be possible to run the different cross validation calculations on different processors and/or threads, allowing the program to crunch through cross validation faster.

Not sure if a CV progress bar would be possible if we did this.

**Comment by tisaconundrum2** [comment]
A CV progress bar would be feasible, in terms of joining threads together.
In other words: say we have 5 processes. As each process finished an emit signal would occur.
so process 
1 of 5, then 2 of 5, 4 of 5 (

**Comment by tisaconundrum2** [comment]
A CV progress bar would be feasible, in terms of joining threads together.
In other words: say we have 5 processes. As each process finished an emit signal would occur.
so process
1 of 5
2 of 5
4 of 5 (because some processes might finish at the same time)
5 of 5 (all processes joined and finished)
This, in essence, could give us a progress bar, however, the problem still remains that there'd be no way to tell how far along each process is.

**Comment by ryanbanderson** [comment]
Yeah, that's how I was hoping it would work. We don't really care how far along each process is, as long as we can track the overall CV progress.
